config/examples/JGAurora/A1/Configuration_adv.h

```diff
@@ -116,7 +116,7 @@
 #define HEATER_BED_INVERTING 1
 #define HEATER_0_INVERTING 1
 //#define HEATER_1_INVERTING 0
-#define FAN_INVERTING 1
+#define FAN_ACTIVE_STATE LOW
```

Comment:
I have a pending PR for this change, but right now it's too soon. At the moment I forget which PR this change is associated with.

Reply:
That wasn’t my commit, so please change what you need to.

The original commits should be good to go.